ScoutSuite/providers/gcp/resources/gce/instances.py

```diff
@@ -0,0 +1,73 @@
+from ScoutSuite.providers.gcp.resources.resources import GCPCompositeResources
+from ScoutSuite.providers.gcp.resources.gce.instance_disks import InstanceDisks
+from ScoutSuite.providers.utils import get_non_provider_id
+
+class Instances(GCPCompositeResources):
+    _children = [ 
+        (InstanceDisks, 'disks')
+    ]
+
+    def __init__(self, gcp_facade, project_id, zone):
+        self.gcp_facade = gcp_facade
+        self.project_id = project_id
+        self.zone = zone
+
+    async def fetch_all(self):
```

Comment:
Indeed 😐

Reply:
Though, since InstanceDisks is a bit different (the disks are already in the Instance, they are not fetched), it's simpler to call `fetch_all()` for each of the disks.